<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prescription_01</class>
 <widget class="QWizardPage" name="Prescription_01">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>415</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string notr="true">Patient</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="lastLabel">
        <property name="toolTip">
         <string comment="PATIENT LAST DISPLAY TOOLTIP">PATIENT LAST</string>
        </property>
        <property name="text">
         <string notr="true">PH PATIENT LAST</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string notr="true">,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="firstLabel">
        <property name="toolTip">
         <string comment="PATIENT FIRST DISPLAY TOOLTIP">PATIENT FIRST</string>
        </property>
        <property name="text">
         <string notr="true">PH PATIENT FIRST</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string notr="true">Medication</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="medicationLabel">
        <property name="toolTip">
         <string comment="MEDICATION DISPLAY TOOLTIP">MEDICATION</string>
        </property>
        <property name="text">
         <string notr="true">PH MEDICATION</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="formLabel">
        <property name="toolTip">
         <string comment="FORM DISPLAY TOOLTIP">FORM</string>
        </property>
        <property name="text">
         <string notr="true">PH FORM</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="strengthLabel">
        <property name="toolTip">
         <string comment="STRENGTH DISPLAY TOOLTIP">STRENGTH</string>
        </property>
        <property name="text">
         <string notr="true">PH STRENGTH</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="unitSizeLabel">
        <property name="toolTip">
         <string comment="UNIT SIZE DISPLAY TOOLTIP">UNIT SIZE</string>
        </property>
        <property name="text">
         <string notr="true">PH UNIT SIZE</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="amountLabel_2">
       <property name="toolTip">
        <string comment="PRESCRIPTION AMOUNT EDIT TOOLTIP">PRESCRIPTION AMOUNT</string>
       </property>
       <property name="text">
        <string comment="PRESCRIPTION AMOUNT EDIT LABEL">PRESCRIPTION AMOUNT</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="amountField">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="PRESCRIPTION AMOUNT EDIT TOOLTIP">PRESCRIPTION AMOUNT</string>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dispenseUnitsLabel">
         <property name="toolTip">
          <string comment="DISPENSE UNITS DISPLAY TOOLTIP">DISPENSE UNITS</string>
         </property>
         <property name="text">
          <string notr="true">PH DISPENSE UNITS</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="writtenLabel">
       <property name="toolTip">
        <string comment="WRITTEN EDIT TOOLTIP">WRITTEN</string>
       </property>
       <property name="text">
        <string comment="WRITTEN EDIT LABEL">WRITTEN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QDateEdit" name="writtenField">
         <property name="toolTip">
          <string comment="WRITTEN EDIT TOOLTIP">WRITTEN</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="filledLabel">
       <property name="toolTip">
        <string comment="FILLED EDIT TOOLTIP">FILLED</string>
       </property>
       <property name="text">
        <string comment="FILLED EDIT LABEL">FILLED</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QDateEdit" name="filledField">
         <property name="toolTip">
          <string comment="FILLED EDIT TOOLTIP">FILLED</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="writtenByLabel">
       <property name="toolTip">
        <string comment="PRESCRIBED BY EDIT TOOLTIP">PRESCRIBED BY</string>
       </property>
       <property name="text">
        <string comment="PRESCRIBED BY EDIT LABEL">PRESCRIBED BY</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="writtenByField">
         <property name="toolTip">
          <string comment="PRESCRIBED BY EDIT TOOLTIP">PRESCRIBED BY</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="filledByLabel">
       <property name="toolTip">
        <string comment="FILLED BY EDIT TOOLTIP">FILLED BY</string>
       </property>
       <property name="text">
        <string comment="FILLED BY DISPLAY LABEL">FILLED BY</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="filledByField">
         <property name="toolTip">
          <string comment="FILLED BY EDIT TOOLTIP">FILLED BY</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>amountField</tabstop>
  <tabstop>writtenField</tabstop>
  <tabstop>filledField</tabstop>
  <tabstop>writtenByField</tabstop>
  <tabstop>filledByField</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
